tests/Test/API.elm

```diff
@@ -1,26 +1,27 @@
 module Test.API exposing (..)
 
-import Expect
-import Test exposing (..)
-import Date exposing (..)
+import Date exposing (Date)
 import DatePicker
     exposing
         ( DatePicker
         , Selection(..)
-        , initCalendar
+        , cancelDates
+        , clearDates
         , getFrom
-        , getTo
-        , getSelectedDate
-        , isOpen
         , getMonth
         , getNextMonth
-        , setDate
-        , toggleCalendar
+        , getSelectedDate
+        , getTo
+        , initCalendar
```

Comment:
guessing this was Elm format alphabetising?

Reply:
Yes, indeed.